tools/deb/istio.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=istio-node-agent: The Istio node agent
+Documentation=http://istio.io/
+
+[Service]
+ExecStart=/usr/local/bin/node-agent-start.sh
```

Comment:
istio-node-agent .... 

(prefix istio because node is confusing - nodejs, etc)